tests/dummy/app/templates/docs/how-it-works.md

```diff
@@ -8,12 +8,12 @@ This addon provides a way to defer work into different paint phases of the rende
 
 ## Concept
 
-Because there isn't a concrete mechanism that allows us to determine when the page is meaningfully painted, it's necessary for us to approximate this. We do so by utilizing a combination of `requestAnimationFrame` calls, which we know have a fairly consistent point of execution (prior to styling, layout, and painting), and scheduling of a macro task using `setTimeout`. Since we know that scheduling a macro task will cause that macro task to be run in the JavaScript event loop _after_ the preceding `requestAnimationFrame` and subsequent paint phase, we can have some fairly dependable guarantees for when work can occur _following a paint_.
+Because there isn't a concrete mechanism that allows us to determine when the page is meaningfully painted, it's necessary for us to approximate this. We do so by utilizing a combination of scheduling an `afterRender` and `requestAnimationFrame` calls, which we know have a fairly consistent point of execution (prior to styling, layout, and painting). Since we know that scheduling successive `requestAnimationFrame` calls following the initial render, we can have some fairly dependable guarantees for when work can occur _following a paint_.
 
 To simply visualize what this looks like in relation to `ember-app-scheduler`'s APIs, this is how we accomplish what's described above:
 
-`requestAnimationFrame` -> Schedule macro task (run.later(fn, 0)) -> browser paint -> macro task runs -> `whenRoutePainted` promise resolves
+`schedule('afterRender')` -> `requestAnimationFrame` -> `requestAnimationFrame` -> browser paint -> `whenRoutePainted` promise resolves
```

Comment:
We can probably remove reference to `whenRoutePainted` in the docs...

Reply:
Sure. Can do.